File: QuickBuy.Dominio/Entidades/Usuario.cs
using System.Collections.Generic;

namespace QuickBuy.Dominio.Entidades
{
    /// <summary>
    /// classe responsável pela definição das propriedades da entidade Usuario.
    /// </summary>
    public class Usuario
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string Senha { get; set; }
        public string Nome { get; set; }
        public string SobreNome { get; set; }
        public ICollection<Pedido> Pedidos { get; set; }
    }
}

File: QuickBuy.Web/ClientApp/src/app/modelo/usuario.ts
/**
 * @file usuario.ts
 * @author: Paulo Alves
 * @description: responsável por definir modelo lógico para a entidade Usuario.
 * @version 1.0.1 (06/10/2020)
 */

/**
 * Classe resposável pela atribuição de propriedades referentes a entidade Usuario.
 */
export class Usuario {
  id: number;
  email: string;
  senha: string;
  nome: string;
  sobrenome: string;
}
